DevAcademy/ContentView.swift

```diff
@@ -2,6 +2,7 @@ import SwiftUI
 
 struct ContentView: View {
     var body: some View {
+<<<<<<< Updated upstream
```

Comment:
tady to vypada na nejaky konflikty... rekl bych, ze bude stacit smazat tu cast `Updated upstream` a nechat tam pouze PlacesScene(), protoze to je novejsi implementace, ktera nahradila tu predchozi.